config/fulcio/fulcio/300-fulcio.yaml

```diff
@@ -73,3 +73,73 @@ spec:
                 - key: ca.crt
                   path: ca.crt
                   mode: 0666
+
+---
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  namespace: fulcio-system
+  name: fulcio-grpc
+spec:
+  template:
+    metadata:
+      annotations:
+        autoscaling.knative.dev/min-scale: "1"
+    spec:
+      serviceAccountName: fulcio
+      # This doesn't actually use Kubernetes credentials, so don't mount them in.
+      automountServiceAccountToken: false
+      containers:
+      - image: gcr.io/projectsigstore/fulcio@sha256:27c6e4fe64a72a537c133452d9c8e0518944d1d69aeee5e7ef8a9fbe70b8b5d3 # v1.0.0
```

Comment:
why not use `gcr.io/projectsigstore/fulcio:v1.0.0@sha256:27c6e4fe64a72a537c133452d9c8e0518944d1d69aeee5e7ef8a9fbe70b8b5d3` and drop the comment